clusters/doks.yaml

```
@@ -24,12 +38,12 @@ releases:
       - default/docker-registry-secrets
     namespace: datadog
     chart: datadog/datadog
-    version: 2.36.4
+    version: 2.36.1
```

```suggestion
    version: 2.36.4
```

Smells like a forgotten `git rebase upstream/main`